common.blocks/popup/popup.tests/gemini.blocks/test-nested/test.styl

```diff
@@ -1 +0,0 @@
-../../simple.blocks/test/test.styl
```

Comment:
So stylus resolves `url`s for background-image properly during the build, but it doesn't do so in case of `@import`s, am I wrong? In other case, I don't understand why there is `../../../../../` in the path